tests/test.rs

```diff
@@ -200,15 +200,24 @@ impl Test {
       command.args(["--shell", "bash"]);
     }
 
-    let mut child = command
-      .args(self.args)
+    command
+      .args(self.args.drain(..))
```

Comment:
What's this drain for?

Reply:
It was needed because now that `self` was behind a mutable reference, we could not do a partial move out of `self` by passing `self.args`.

This is gone with the revert of `Test::run()` split.